apps/hobbies-helsinki/tsconfig.json

```diff
@@ -33,6 +33,8 @@
     "**/*.cjs",
     "**/*.mjs",
     "**/*.jsx",
-    "**/*.json"
+    "**/*.json",
+    "src/domain/app/headlessCmsHelper.tsx",
```

Comment:
why?

Reply:
some reason why the vscode adds to automatically. It's an accident. Nice catch!